src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/kazaSwerveModule/KazaSwerveModule.java

```diff
@@ -33,56 +31,44 @@ public KazaSwerveModule (SwerveChassis.Module module){
         boolean linInverted;
         this.module = module;
 
-        SwerveModuleConfigObject configObject;
-
-        switch (module){
-            case BACK_RIGHT:
-                configObject = KazaSwerveConstants.KazaModuleBackRight;
-                break;
-            case BACK_LEFT:
-                configObject = KazaSwerveConstants.KazaModuleBackLeft;
-                break;
-            case FRONT_RIGHT:
-                configObject = KazaSwerveConstants.KazaModuleFrontRight;
-                break;
-            case FRONT_LEFT:
-                configObject = KazaSwerveConstants.KazaModuleFrontLeft;
-                break;
-            default:
-                throw new IllegalArgumentException("Invalid module");
-        }
+        SwerveModuleConfigObject configObject = switch (module) {
+            case BACK_RIGHT -> KazaSwerveConstants.KAZA_SWERVE_MODULE_BACK_RIGHT;
+            case BACK_LEFT -> KazaSwerveConstants.KAZA_SWERVE_MODULE_BACK_LEFT;
+            case FRONT_RIGHT -> KazaSwerveConstants.KAZA_SWERVE_MODULE_FRONT_RIGHT;
+            case FRONT_LEFT -> KazaSwerveConstants.KAZA_SWERVE_MODULE_FRONT_LEFT;
+            default -> throw new IllegalArgumentException("Invalid module");
```

Comment:
Switch it for invalid swerve module so in the future if we will use module as a name for something else it will be clear